addon/components/mobiledoc-editor/component.js

```diff
@@ -217,6 +220,7 @@ export default Component.extend({
         this.get('componentAtoms').removeObject(atom);
       }
     };
+    editorOptions.cardOptions = assign(componentHooks, editorOptions.cardOptions);
```

Comment:
@kylenathan we're not convinced that merging user cardOptions over the editor built-in hooks is best. Really the built-in hooks should not need to be stomped, and if they are stomped we will end up with a mess.

At some point we should make the `ADD_CARD_HOOK` constant into a symbol, since we don't want the property name to be guessable by the outside world, and don't want it to conflict either.

This can just be reversed though, and the componentHooks can win. I'm going to make that change and land this 🎉